contrib/golang/filters/http/source/golang_filter.cc

```diff
@@ -1154,6 +1154,12 @@ FilterConfig::FilterConfig(
   ENVOY_LOG(debug, "golang filter new plugin config, id: {}", config_id_);
 };
 
+FilterConfig::~FilterConfig() {
+  if (config_id_ > 0) {
+    dso_lib_->envoyGoFilterDestroyHttpPluginConfig(config_id_);
```

Comment:
If only the Go filter config changes, does the merged_config_id configuration also need to be deleted?

Reply:
yep, there is a old bug for the `merged_config_id_` cache, already fixed, thanks!